### PR TITLE
removing half developed key send

### PR DESCRIPTION
**Pull request by nicolasburtey**
will be reimplemented later with AMP

### CODE REVIEW

File: src/lndUtils.ts [hunk 1]
```diff
@@ -469,17 +457,15 @@ export const validate = async ({
...
     username = params.username
+  }
 
-    const preimage = randomBytes(preimageByteLength)
-    id = createHash("sha256").update(preimage).digest().toString("hex")
-    const secret = preimage.toString("hex")
-    messages = [{ type: keySendPreimageType, value: secret }]
+  if (!!params.amount && !!tokens) {
+    const error = `Invoice contains non-zero amount, but amount was also passed separately`
+    // FIXME: create a new error. this is a not a graphl error.
```
Comment (krtk6160) [inline src/lndUtils.ts:465]: We already have a `ValidationError` in the `error.ts`. It looks like the wrong one was imported here in this file.

File: src/lndUtils.ts [hunk 2]
```diff
@@ -444,52 +439,38 @@ export const validate = async ({
...
+    // throw new ValidationInternalError(error, {logger})
```
Comment (bodymindarts) [inline src/lndUtils.ts:462]: This comment is duplicate of the line below??

**Review by bodymindarts — APPROVED** [review]
LGTM I'm guessing most of the formatting and adding type information were in order to get rid of linting errors right?

File: src/lndUtils.ts (hunk 2 above)
Comment (nicolasburtey) [reply src/lndUtils.ts:462]: no the condition are different. it's basically an xor we are doing here, which an explicit error on whether we have too much info or too little

**Pull request merged**